src/Agent/NewRelic/Agent/Core/Logging/AuditLog.cs

```diff
@@ -5,15 +5,15 @@ namespace NewRelic.Agent.Core.Logging
 {
     public static class AuditLog
     {
-        private static readonly log4net.ILog Logger = log4net.LogManager.GetLogger(typeof(AuditLog));
-
         /// <summary>
         /// Logs <paramref name="message"/> at the AUDIT level, a custom log level that is not well-defined in popular logging providers like log4net. This log level should be used only as dictated by the security team to satisfy auditing requirements.
         /// </summary>
         public static void Log(string message)
         {
-            var auditLogLevel = LoggerBootstrapper.GetAuditLevel();
-            Logger.Logger.Log(typeof(AuditLog), auditLogLevel, message, null);
+            // create a logger that injects an Audit property that we can later filter on (see LoggerBootstrapper.IncludeOnlyAuditLog)
+            var auditLogger = Serilog.Log.ForContext(LogLevelExtensions.AuditLevel, LogLevelExtensions.AuditLevel);
```

Comment:
Do we need to cache this logger as a static field? Is the `ForContext` call expensive?

Reply:
Actually, there's a better way to do this at the log config level, I'll make the change.

Reply:
Resolved this with a `Lazy<>` - the config-level change I wanted to make didn't work out.